backend/core/Comparers/DeepPropertyCompare.cs

```diff
@@ -24,10 +24,11 @@ public new bool Equals([AllowNull] object x, [AllowNull] object y)
         {
             // Only root objects cannot be null.  Child objects that are null will be equal.
             if (x == null || y == null || GetHashCode(x) != GetHashCode(y) || x.GetType() != y.GetType()) return false;
+            if (x == y) return true;
 
             var type = x.GetType();
             var children = type.GetCachedProperties(BindingFlags.Public | BindingFlags.Instance | BindingFlags.GetProperty).Where(p => !p.PropertyType.IsValueType && p.PropertyType != typeof(string) && !p.PropertyType.IsEnumerable());
-            foreach (var p in children)
+            foreach (var p in children.Where(p => !p.GetIndexParameters().Any()))
```

Comment:
The new GIS objects were causing issues when doing equal comparison tests.  This isn't an ideal solution, but appears to work for our use-case.

Reply:
I think I would prefer to explicitly choose to ignore problematic fields rather then silently ignore everything that is not indexed. I worry this approach will lead to false positives, as anyone who isn't familiar with the above may not realize that it is ignoring some fields.